src/templates/prototypes/front-page.pug

```diff
@@ -4,16 +4,19 @@ block content
 
   .section--primary.hero(tabindex=0)#content
     .container
-      h1 Welcome to the UK Parliament test website
+      h1 Welcome to UK Parliament's beta website
+
+      p We're building a #[+link('#', 'new website for Parliament')]. Pages with a beta label are being tested and improved. They may not work for everyone.
+      p Beta pages are separate from the current parliament.uk website
+
       p We're building a #[+link('#', 'new website for Parliament')]. These pages are being tested and improved - they may be inaccurate or misleading.
```

Comment:
Are lines 12 & 13 duplicates?

Reply:
The front page content is not the same as on beta, just a rough estimate to check visually how things look. Not to worry about this as it is just a prototype. The elements views would be more important as it contains our components.